data/data-pipeline/data_pipeline/score/field_names.py

```diff
@@ -564,6 +564,63 @@
     f"Low high school education{ISLAND_AREAS_SUFFIX}"
 )
 
+# Booleans for percentile exceeded thresholds
```

Comment:
These are all new field names for these booleans. 

Field names is currently pretty clunky but I think we can wait to the refactor to streamline it :)